package.json

```diff
@@ -111,4 +112,4 @@
   "browserslist": [
     "defaults"
   ]
-}
+}
```

Comment:
:(

Reply:
hehe I don't know why but always happens on `npm i`

Reply:
think I've looked it up before but forget

Reply:
think I've looked it up before but forget